client-sdk/go/types/crypto.go

```diff
@@ -71,3 +72,69 @@ func (pk *PublicKey) UnmarshalJSON(data []byte) error {
 	}
 	return pk.unmarshal(&spk)
 }
+
+// MultisigSigner is one of the signers in a multisig configuration.
+type MultisigSigner struct {
+	PublicKey PublicKey `json:"public_key"`
+	Weight    uint64    `json:"weight"`
+}
+
+// MultisigConfig is a multisig configuration.
+// A set of signers with total "weight" greater than or equal to a "threshold" can authenticate
+// for the configuration.
+type MultisigConfig struct {
+	Signers   []MultisigSigner `json:"signers"`
+	Threshold uint64           `json:"threshold"`
+}
+
+// Verify performs some sanity checks.
+func (mc *MultisigConfig) Verify() error {
```

Comment:
Should we call this `Validate` or `ValidateBasic` instead?

Reply:
This should be called anything but `Verify` probably.

Reply:
ok ValidateBasic